.changeset/pistachio-poultry-fish.md

```diff
@@ -0,0 +1,5 @@
+---
+'@finos/legend-graph': patch
+---
+
+Add DND for class properties ([#303] https://github.com/finos/legend-studio/issues/303
```

Comment:
I would leave this changeset empty. The DND feature is in legend-studio.

Reply:
makes sense, thanks!